chrome/content/coniks/coniks.js

```diff
@@ -0,0 +1,118 @@
+this.EXPORTED_SYMBOLS = ["coniks"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("resource:///modules/imServices.jsm");
+Cu.import("chrome://otr/content/coniks/context.js");
+Cu.import("chrome://otr/content/helpers.js");
+Cu.import("chrome://otr/content/otr.js");
+
+function accountname(acc) {
+  let account, protocol;
+  if (acc.normalizedName !== undefined &&
+    acc.protocol.normalizedName !== undefined) {
+    account = acc.normalizedName;
+    protocol = acc.protocol.normalizedName;
+  } else {
+    account = acc.account;
+    protocol = acc.protocol;
+  }
+  return `${account}@${protocol}`;
+}
+
+var coniks = {
+  isEnable: false,
+
+  init: function() {
+    coniks.setPrefs();
+    coniks.isEnabled = coniks.prefs.getBoolPref("enabled");
+    if (!coniks.isEnabled)
+      return;
+
+    this._contexts = new Map();
+    for (let acc of helpers.getAccounts()) {
+      coniks._contexts.set(accountname(acc), new AccountContext({
+        account: acc.normalizedName,
+        protocol: acc.protocol.normalizedName
+      }));
+    }
+    return coniks.loadFiles();
```

Comment:
I think we would load more files in the future, so maybe it should still be kept in a separate function `loadFiles`?